src/Routes.jsx

```diff
@@ -12,7 +12,7 @@ const router = App => (
   <Router history={history}>
     <Switch>
       <Route path="/" exact component={Home} />
-      <Route path="/add" exact component={Add} />
+      <Route path="/add" exact component={Add} history={history}/>
```

Comment:
The `history` will be passed to `component` in `Route` Component implicitly.

Reply:
ok